### DISCUSSION TITLE
Import playbooks and inventories from standard directory structure

### DISCUSSION THREAD

**Discussion by salvioli** [semaphoreui/semaphore#938] [Q&A]
Before looking into the ansible semaphore documentation I was expecting to be able to bring in my ansible project and potentially with some minor re-arrangement of the ansible folder structure I already have in place semaphore can automatically import all playbooks and inventories for me.

It seems that this is not the case from the documentation, is there a way to import all data from filesystem and avoid to use REST API or the web gui for setting up the node?

**Comment by fiftin** [comment]
Which directory structure is standard?

**Reply by salvioli** [reply]
I agree none is standard. Nevertheless the project may define a standard project structure that the user may comply with. 
Though I guess I was thinking more about something like the user providing a directory root that the tool can scan for playbooks, inventories, roles, etc... 

PS: Excuse me if I am being too off with respect to the current tool behaviour but some time has passed since I tried to use it and both the tool may have evolved as well as I have forgotten how it behaved.
